components/eamxx/src/share/tests/property_checks.cpp

```diff
@@ -205,6 +205,27 @@ TEST_CASE("property_checks", "") {
     interval_check->repair();
     res_and_msg = interval_check->check();
     REQUIRE(res_and_msg.result==CheckResult::Pass);
+
+    // Re-assign an out-of-bounds value to the field, but only in the upper-bound.  Check if it
```

Comment:
Should "upper" here be "lower" in the comment?

Reply:
oh yes, copy/paste error.